groups/bdl/bdlb/bdlb_guid.h

```diff
@@ -517,6 +521,13 @@ unsigned char Guid::version() const {
     return uc(timeHiAndVersion() >> 12);
 }
 
+template <class HASH_ALGORITHM>
+void hashAppend(HASH_ALGORITHM& hashAlg, const Guid& guid)
```

Comment:
FWIW, the coding standards has this: "Inline free operators are defined outside of the package namespace in order to prevent inadvertent self-declaration."

Which I believe means, to avoid an accident like:

    class A {
    };
    
    bool operator==(const A& lhs, const &rhs);
    
     // INLINE DEFINITIONS
    inline bool operator==(const A lhs, const A& rhs)
    {
       // ...
    }
       // oops... this definition is actually a separate declaration but it compiles

Does that apply to free functions as well as operators? Probably. Does that kind of mistake happen very often? I think I've seen it once...